gulpfile.ts

```diff
@@ -1,5 +1,9 @@
+import SwaggerParser from '@apidevtools/swagger-parser' // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
To fix the `import/no-extraneous-dependencies` rule, please make a PR to the `utils` repo, and add `gulpfile.ts` to the array of file names:

https://github.com/ScaleLeap/utils/blob/5782c814ba042c019ab65957f26a89051cc9021e/eslint/index.js#L164

Once `utils` package release, you can update it and the warning will be gone.